### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by OscarArdila15** [codesandbox/codesandbox-client#4553]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 83.0.4103 | Windows 10           | c0176a2d4          |

  **Route:**
  https://codesandbox.io/s/late-cdn-kwkbk?file=/src/index.js

  

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
  Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at ys (https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1756798)
    at ms (https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1754799)
    at ys (https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1756535)
    at yl (https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1775800)
    at https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1797332
    at qo (https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1721705)
    at bl (https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1773685)
    at il (https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1764012)
    at https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1721996
    at https://codesandbox.io/static/js/common.e7a97ddf5.chunk.js?_sw-precache=c00dd7bed9d861d67b93fee2a4434e77:1:1797332
  
    in elements__Container
    in div
    in Ul
    in div
    in Styled(styled.div)
    in li
    in Entry___StyledListAction
    in div
    in n
    in div
    in Unknown
    in DragSource(Component)
    in div
    in elements__EntryContainer
    in div
    in Unknown
    in DropTarget(Component)
    in div
    in Is
    in div
    in elements__Opener
    in div
    in Unknown
    in DropTarget(Component)
    in div
    in Is
    in div
    in elements__Opener
    in div
    in Unknown
    in DropTarget(Component)
    in div
    in Styled(styled.div)
    in section
    in Styled(styled.div)
    in Unknown
    in Files___StyledCollapsible
    in $s
    in Ys
    in div
    in je
    in Unknown
    in div
    in elements__Container
    in Unknown
    in WithTheme(Component)
    in div
    in t
    in div
    in t
    in div
    in div
    in styled.div
    in div
    in elements__Container
    in je
    in Unknown
    in Unknown
    in Unknown
    in t
    in t
    in div
    in elements__Content
    in n
    in withRouter(n)
    in div
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in je
    in c
    in t
  ```

  </details>

**Comment by oskarabc** [comment]
Hello @OscarArdila15 can you please post a few more steps, before you it crashed?

**Comment by SaraVieira** [comment]
closed by #4571 4571
